<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>974</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>721</width>
     <height>961</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>660</y>
     <width>661</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(23, 156, 145);
	color: rgb(76, 141, 131);
	color:rgb(0,0,0);
	background-color:rgb(58, 76, 168);
	color: rgb(255, 255, 255);
	border:0px solid;
	border-radius:10px;
}
QPushButton:hover{
	background-color:rgb(23, 156, 145);
}
</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>661</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(23, 156, 145);
	color: rgb(76, 141, 131);
	color:rgb(0,0,0);
	background-color:rgb(58, 76, 168);
	color: rgb(255, 255, 255);
	border:0px solid;
	border-radius:10px;
}
QPushButton:hover{
	background-color:rgb(23, 156, 145);
}
</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>790</y>
     <width>661</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(23, 156, 145);
	color: rgb(76, 141, 131);
	color:rgb(0,0,0);
	background-color:rgb(58, 76, 168);
	color: rgb(255, 255, 255);
	border:0px solid;
	border-radius:10px;
}
QPushButton:hover{
	background-color:rgb(23, 156, 145);
}
</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
